rotkehlchen/db/filtering.py

```diff
@@ -1092,7 +1093,15 @@ def make(
             filter_query.filters.append(DBMultiBytesFilter(
                 and_op=True,
                 column='tx_hash',
-                values=tx_hashes,  # type: ignore  # EVMTxHash is equal to bytes
+                values=cast(list[bytes], tx_hashes),  # EVMTxHash is equal to bytes
```

Comment:
keep the tx ignore. As cast is kind of silly and is an actual function also in production: https://docs.python.org/3/library/typing.html#typing.cast
```suggestion
                values=tx_hashes,  # type: ignore # EVMTxHash is equal to bytes
```

I mean it's super minimal, but why put it here when comment + type: ignore achieve the same thing. I don't want us to add extra code just to just appease mypy